docs/developer/devicesimulator.md

```
@@ -1,22 +1,295 @@
 ### Setup
 
 Copy the necessary setup files to run a `devicesimulator`.
 
-    mkdir -p ~/.flightctl/certs/ ~/.config/flightctl/
+    mkdir -p ~/.flightctl/certs/ ~/.config/flightctl/ 
     cp bin/agent/etc/flightctl/certs/* ~/.flightctl/certs/
     cp bin/agent/etc/flightctl/config.yaml ~/.flightctl/agent.yaml
 
+Changes to the configuration of all launched agents can be made in `~/.flightctl/agent.yaml`
+Some key options available for tuning:
+- `spec-fetch-interval` - Affects how often the agent queries the API. Higher intervals reduce load on the server.
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)